app/cafe.py

```diff
@@ -0,0 +1,26 @@
+import datetime
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        visitor_name = visitor.get("name", "Unknown")
+
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError(visitor_name)
+
+        expiration_date = visitor["vaccine"].get("expiration_date")
+        if expiration_date < datetime.date.today():

```

Comment:
You should ensure that `expiration_date` is not None before comparing it with today's date. Consider adding a check to handle cases where `expiration_date` might be missing.